### PR TITLE
Sorr task64 refine first review process doc

### PR DESCRIPTION
**Pull request by DanilYachmenev**
#64 

Refined First review doc with some additional processes that we agree to be good practise

### CODE REVIEW

**Comment by DanilYachmenev** [comment]
Merging to speed up the process and to be able to reference it to the students
Change seems to be small and agreed before
If any change is needed, will do in a follow-up

File: docs/First_review_process.md [hunk 1]
```diff
@@ -69,6 +70,24 @@ some helpful tips and resources to guide you through your first review.
     - `defi/tulip/test/test_order_matching.py`
     - `defi/tulip/test/test_optimize.py`
 
+## Save reviewers time
+
+- Make sure to assign the PR to the reviewers so they get notified
+  - <img width="313" alt="Снимок экрана 2023-05-29 в 03 51 01" src="https://github.com/sorrentum/sorrentum/assets/31514660/f8534c49-bff6-4d59-9037-d70dc03d5ff9">
+  - Juniors should assign Team Leaders to review their PR
```
Comment (gpsaggese) [inline docs/First_review_process.md:77]: -> Junior contributors 
-> Team Leaders (e.g., Grisha, DanY, Samarth, ...)

File: docs/First_review_process.md [hunk 2]
```diff
@@ -69,6 +70,24 @@ some helpful tips and resources to guide you through your first review.
...
+    - Team Leaders will assign integrators (GP & Paul) themselves after all their comments are implemented
+  - Ping the assigned reviewer in the issue if nothing happens in 24 hours
+- Mention the corresponding issue in the PR description to ease the navigation
+  - E.g., see an [example](https://github.com/sorrentum/sorrentum/pull/288#issue-1729654983)
+    - <img width="505" alt="Снимок экрана 2023-05-29 в 03 51 29" src="https://github.com/sorrentum/sorrentum/assets/31514660/69fbabec-300c-4f7c-94fc-45c5da5a6817">
```
Comment (gpsaggese) [inline docs/First_review_process.md:82]: Не могли бы вы удалить русский комментарий здесь?

File: docs/First_review_process.md [hunk 3]
```diff
@@ -69,6 +70,24 @@ some helpful tips and resources to guide you through your first review.
...
+- When you've implemented a comment from a reviewer, press `Resolve conversation` button so the reviewers know that you actually took care of it
```
Comment (gpsaggese) [inline docs/First_review_process.md:83]: -> Reviewer (everywhere)

**Review by gpsaggese — COMMENTED** [review]
Good stuff. Minor tweaks

**Pull request merged**